docs/_includes/infotip.html

```diff
@@ -13,7 +14,7 @@
                     </svg>
                 </button>
                 <div class="infotip__overlay">
-                    <span class="infotip__pointer infotip__pointer--top-left"></span>
+                    <span data-popper-arrow></span>
```

Comment:
Wouldn't this be a breaking change?

Reply:
Because this is only a docs change. The old code is still available for teams to use.

Reply:
Oh, right. Cool.

Reply:
It won't be a breaking change for skin, but if we do this in eBayUI too we should tread lightly because there are a _lot_ of teams who override behavior of popovers to make them open in different directions.

Reply:
If they do that that's on them. They shouldn't be overriding base behavior 
But we do allow style overrides there. We probably should do something like
If they have a style override don't use popper to position.
If not then use popper.

Reply:
I would prefer we not go down the road of checking for overrides and only using popper when none exist. I'd rather avoid various cases based on specific conditions if we can help it. If it comes down to it, I would actually prefer releasing this in a major version and warning people to remove overrides as part of the SWU.

Reply:
So I think there is a bit of misconception. First off this change would be only in ebayui. In skin there is no issue. In fact the old code works as is no problem (see storybook). 
This would only affect ebayui.

For ebayui I never said to not use popper if we have style overrides. We just don't use poppers automatic positioning. 
Popper allows you to override the position and setting offsets. What we would do in that case if they pass styles is to pass the position changes to popper and that would take care of it. 
So there wouldn't be anything breaking about it. It only only change internal implementation.

Reply:
This is probably technically not a breaking change. The offsets are probably not an issue either; they serve as slight nudges. The positioning and overrides are the main thing I think.

Would we automatically send their styles into Popper without any checks? Or would we need to filter out the positioning ones and only send those to Popper? Would there be any sort of conditional overhead inside? If they override Popper's positioning, then the onus is on them to maintain it, right?

The main thing, I'd like to avoid is unnecessary positioning considerations being handled in ebay ui that later needs to be deprecated.

It's a bit hard conceptualizing this. Maybe if we had the ebay ui implementation, it might be easier to see if this poses any potential issue.